.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea/
 *.log
+*.db
+invoice_preview.html
```

Comment:
this is an automatically generated file? is this why we're ignoring it?

Reply:
Right.

Reply:
Cool thanks for clarifying.